usaspending_api/api_contracts/contracts/v2/reporting/agencies/agencies.md

```diff
@@ -0,0 +1,101 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agencies Reporting Overview [/api/v2/reporting/agencies/overview?{fiscal_year,fiscal_period,search,page,limit,order,sort}]
+
+This endpoint is used to power USAspending.gov's about the data agencies page. This data can be used to better understand the ways agencies submit data.
+
+## GET
+
+This endpoint returns an overview of government agencies submission data.
+
++ Parameters
+
+    + `fiscal_year`: 2020 (required, number)
+        The fiscal year.
+    + `fiscal_period`: 10 (required, number)
```

Comment:
@tony-sappe front end will just send the last period for a given quarter for pre-2020? Or should we have. a new param called `fiscal_quarter`?

Reply:
I assumed the last period of a quarter would be sufficient. Isn't that how things are done for Spending Explorer and Custom Account Downloads?

Reply:
edit: ha, guess not. Can you think of any issues if only periods are used for older submissions?

Reply:
Not really except for inconsistent behavior across multiple endpoints. `/submission_peroids`. Sending the latest period is the right thing to do IMO.